scripts/post-update.sh

```diff
@@ -360,6 +360,22 @@ if (( $PRE_0_0_22 > 0 )); then
     sudo python3 $HOME/companion/services/network/setup.py install
 fi
 
+PRE_0_0_23=$(( git rev-list --count --left-right 0.0.23...revert-point || echo 0 ) | cut -f1)
+
+if (( $PRE_0_0_23 > 0 )); then
+    # reconfigure mavproxy to reuse the same logfile, and set the autopilot to use mavlink2
+    # reconfigure only if a user configuration exists in the home directory
+    USER_MAVPROXY_PARAMS=$HOME/mavproxy.param
+    if [ -e $USER_MAVPROXY_PARAMS ]; then
+        # delete any streamrate line if it already exists
+        sed -i '/--aircraft*/d' $USER_MAVPROXY_PARAMS
+        # append the streamrate line we require
+        sed -i '$ a --cmd="param set SERIAL0_PROTOCOL 2"' $USER_MAVPROXY_PARAMS
+    fi
+    # remove old logs (free disk space)
+    rm -r $HOME/telemetry
```

Comment:
maybe add -f to be sure that everything will be removed and the command will not fail.